apps/web-giddh/src/app/daybook/daybook.component.ts

```diff
@@ -376,58 +378,80 @@ export class DaybookComponent implements OnInit, OnDestroy {
     public hideExportDaybookModal(response: any) {
         this.modalDialogRef.close();
         if (response !== 'close') {
```

Comment:
```suggestion
        if (response && response !== 'close') {
```

Reply:
This code is not written by me it was written by someone five years ago.